libraries/bsg_manycore_config.h

```diff
@@ -104,8 +107,7 @@ extern "C" {
                 uint32_t io_endpoint_max_out_credits;
                 uint32_t dram_channels;
                 uint32_t dram_bank_size_words;
```

Comment:
Should these be part of the memory system too?

Reply:
Seems reasonable.